src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs !!!
+    internal class FeatureFlag
+    {
+        private const string Prefix = "VSTEST_FEATURE_";
+
+        public Dictionary<string, bool> FeatureFlagsBag { get; } = new();
+
+        public static FeatureFlag Default { get; } = new FeatureFlag();
+
+        public FeatureFlag()
+        {
+            FeatureFlagsBag.Add(ARTIFACTS_POSTPROCESSING, false);
+        }
+
+        // Added for artifact porst-processing, it enable/disable the post processing.
+        // Added in 17.2-preview 7.0-preview
+        public const string ARTIFACTS_POSTPROCESSING = Prefix + "ARTIFACTS_POSTPROCESSING";
```

Comment:
`ARTIFACTS_POSTPROCESSING_SDK_KEEP_OLD_UX` is missing

Reply:
Not needed here we're not using it inside the SDK. We need to keep in sync only used flag